pkg/controller/cronjob/cronjob_controller.go

```diff
@@ -35,6 +35,7 @@ import (
 
 	"k8s.io/klog/v2"
 
+	"github.com/robfig/cron"
```

Comment:
Can the changes to existing controller be factored out to a separate PR?
I would ideally like to see this PR just adding new controller and not touch any existing code.

Reply:
addressed this